### PR TITLE
Simple_logger now logs protobuf ref packets

### PR DESCRIPTION
**Pull request by ryanstrat**
There are still further discussions regarding how to properly include filtered vision data, but as far as updating the existing simple logger for compatibility, the work has been completed.

It is also now capable of taking vision data in from the simulator with the -sim flag

### CODE REVIEW

File: logging/simple_logger.cpp [hunk 1]
```diff
@@ -95,21 +122,31 @@ int main(int argc, char* argv[]) {
                 printf("Bad vision packet of %d bytes\n", n);
                 continue;
             }
+
+            // LogFrame_Robot* robot = logFrame
```
Comment (justbuchanan) [inline logging/simple_logger.cpp]: x

**Comment by justbuchanan** [comment]
Looks good to me.

**Comment by ryanstrat** [comment]
@jgkamat Am I safe to merge this? CI is still having issue, right?

**Comment by jgkamat** [comment]
You can go ahead and merge it (the issues were fixed on master, but it dosent matter since non-style is passing)

**Pull request merged**